src/escpos/escpos.py

```diff
@@ -733,6 +738,18 @@ def panel_buttons(self, enable=True):
         else:
             self._raw(PANEL_BUTTON_OFF)
 
+    def queryStatus(self):
+        self._raw(RT_STATUS_ONLINE)
+        time.sleep(1)
+        status = self._read();
+        if (len(status) > 0):
+            return status;
+        else:
+            return [8];
+
+    def isOnline(self):
+        status = self.queryStatus()[0];
+        return ((status & RT_MASK_ONLINE) == 0);
```

Comment:
Oops, sorry, you're right, the operator must be '&'. I'm not familiar with bitwise operators.

Is this the only way to return the status boolean?
Maybe it could work ``return bool(status is [8])`` or similar?